app/back/create/template.hbs

```diff
@@ -0,0 +1,10 @@
+<div class="create-page">
+    <form {{action 'save' model on='submit'}}>
+    <label for="menu-item">Menu Item:</label>
+    {{input id="menu-item" type="text" value=model.menuItem}}
+
+    <label for="menu-price">Price:</label>
+    {{input id="menu-price" type="text" value=model.price}}
+    </form>
+    <button>{{#link-to 'back.read'}}Add Item to Menu{{/link-to}}</button>
```

Comment:
You don't need a link to. You want the form to submit not moving to something else.